src/argilla/client/feedback/integrations/textdescriptives.py

```diff
@@ -43,9 +44,8 @@ def __init__(
         self,
         model: str = "en",
         metrics: Optional[List[str]] = None,
-        fields: Optional[List[str]] = None,
-        visible_for_annotators: bool = True,
-        show_progress: bool = True,
+        visible_for_annotators: Optional[bool] = True,
```

Comment:
@gabrielmbmb, do you think we should also move the `metrics` and `visible_for_anotators` to the `update_records` and `update_dataset` methods?